.vscode/tasks.json

```diff
@@ -1,48 +1,50 @@
 {
-	"version": "2.0.0",
-	"presentation": {
-		"echo": true,
-		"reveal": "always",
-		"focus": false,
-		"panel": "shared"
-	},
-	"tasks": [
-		{
-			"label": "build",
-			"command": "npm",
-			"type": "shell",
-			"args": [
-				"run",
-				"compile"
-			],
-			"group": {
-				"kind": "build",
-				"isDefault": true
-			}
-		},
-		{
-			"label": "buildDev",
-			"command": "npm",
-			"type": "shell",
-			"args": [
-				"run",
-				"compileDev"
-			],
-			"group": {
-				"kind": "build",
-				"isDefault": true
-			}
-		},
-		{
-			"label": "test",
-			"command": "echo",
-			"group": {
-				"kind": "test",
-				"isDefault": true
-			},
-			"args": [
-				"Run tests in VS Code by launching the debugger with the 'Launch Tests' configuration."
-			]
-		},
-	]
-}
\ No newline at end of file
+    "version": "2.0.0",
+    "presentation": {
+        "echo": true,
+        "reveal": "always",
+        "focus": false,
+        "panel": "shared"
+    },
+    "tasks": [
+        {
+            "label": "build",
+            "command": "npm",
+            "type": "shell",
+            "args": ["run", "compile"],

```

Comment:
Recommend whitespace off for this. Formatter fixed some indentation and normalized tabs/spaces